<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotMembers</class>
 <widget class="QWidget" name="PlotMembers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot PDF members</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resource.qrc">
    <normaloff>:/icons/logo.png</normaloff>:/icons/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/splash.png);
background-repeat: no;
background-position: center;
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Physical setup</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="7" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="setmember">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QCheckBox" name="automaticrange">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Automatic range selection</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="stddev">
         <property name="text">
          <string>Std. deviation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="cl">
         <property name="text">
          <string>68% c.l.</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QRadioButton" name="lin">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Linear scale</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>x min.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="xmin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>1e-5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="xmax">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>x max</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>y min.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="ymin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>-10</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="ymax">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>10</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>y max</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Plot all members</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="Qf">
         <property name="text">
          <string>1.4142135623731</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="xpoints">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000000000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="Qi">
         <property name="text">
          <string>1.4142135623731</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>PDF flavor:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Initial scale (GeV):</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Final scale (GeV):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Select member:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QRadioButton" name="log">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Logarit. scale</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="PDFflavor">
         <property name="currentIndex">
          <number>6</number>
         </property>
         <item>
          <property name="text">
           <string>tbar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>bbar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cbar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>sbar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ubar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dbar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gluon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>u</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>c</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>b</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>t</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>photon</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Number x points:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="mean">
         <property name="text">
          <string>Mean value</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Plot editor</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Plot Title:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>X axis label:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Y axis label:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Replica color:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="QComboBox" name="colorreplica">
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Violet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Magenta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gray</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Central value color:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QComboBox" name="coloravg">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Violet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Magenta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gray</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Std. Dev. color:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QComboBox" name="colorstddev">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Violet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Magenta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gray</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>68% c.l. color:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="QComboBox" name="color68cl">
         <property name="currentIndex">
          <number>6</number>
         </property>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Orange</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Violet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Magenta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Black</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cyan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gray</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="11" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>59</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLineEdit" name="title"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLineEdit" name="xtitle"/>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QLineEdit" name="ytitle"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="playButton">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Compute</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resource.qrc">
       <normaloff>:/images/StepForwardNormalBlue.png</normaloff>:/images/StepForwardNormalBlue.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="saveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Save plot</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resource.qrc">
       <normaloff>:/images/save-32x32.png</normaloff>:/images/save-32x32.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
